tests/e2e/Services/Storage/StorageBase.php

```diff
@@ -145,7 +145,7 @@ public function testCreateBucketFile(): array
         ];
         $id = '';
         $curlFile = new \CURLFile('data://' . $mimeType . ';base64,' . base64_encode(@fread($handle, $chunkSize)), $mimeType, 'large-file.mp4');
-        $headers['content-range'] = 'bytes ' . ($counter * $chunkSize) . '-' . min(((($counter * $chunkSize) + $chunkSize) - 1), $size) . '/' . $size;
+        $headers['content-range'] = 'bytes ' . ($counter * $chunkSize) . '-' . min(((($counter * $chunkSize) + $chunkSize) - 1), $size - 1) . '/' . $size;
```

Comment:
doesn't this need to be updated in our SDKs as well ? 
https://github.com/appwrite/sdk-for-php/blob/master/src/Appwrite/Services/Storage.php#L391 
and all the other SDKs too